processing/data_collection/gazette/spiders/sp_fernandopolis.py

```diff
@@ -0,0 +1,35 @@
+from dateparser import parse as parse_date
+from datetime import datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpFernandopolis(BaseGazetteSpider):
+    MUNICIPALITY_ID = "3515509"
+    name = "sp_fernandopolis"
+    start_urls = [
+        "https://diariotransparente.com.br/publicacoes/frame/spfernandopolispm"
+    ]
+
+    def parse(self, response):
+        IDX_EXTRA_EDITION_LI = 3
+        list_pubs = response.css("div.row")
+        for pub in list_pubs:
+            link_date = pub.css("h3>a")
+            date = link_date.css("::text").get()
+            date = parse_date(date, languages=["pt"]).date()
+            link = response.urljoin(link_date.attrib["href"])
+            is_extra_edition = (
```

Comment:
**issue (blocking)**: Extra edition is not being collected correctly. Extra editions are assigned as `False` instead of `True` (and non-extra are inverted).

**suggestion**: You can check the presence of `EXTRA` string in the `<li>` tag:

`is_extra_edition = bool(extra_pub.xpath(".//li[contains(., 'EXTRA')]"))`

Reply:
I was searching for "EXTRA" string in `li` element, but I inverted the boolean values:
Wrong code: `pub.css("ul>li::text")[IDX_EXTRA_EDITION_LI].re_first("EXTRA") is None`;
Correct: `pub.css("ul>li::text")[IDX_EXTRA_EDITION_LI].re_first("EXTRA") is not None`.

I fixed the code like you suggest.